crates/wasmcloud-host/src/host.rs

```diff
@@ -147,18 +149,18 @@ impl HostBuilder {
     /// accidental mutation of images, close potential attack vectors, and prevent against
     /// inconsistencies when running as part of a distributed system. Also keep in mind that if you
     /// do enable images to run as 'latest', it may interfere with live update/hot swap
-    /// functionality
+    /// functionality.
     pub fn oci_allow_latest(self) -> HostBuilder {
         HostBuilder {
             allow_latest: true,
             ..self
         }
     }
 
-    /// Allows the host to pull actor and capability provider images from these registries without
-    /// using a secure (SSL/TLS) connection. This option is empty by default and we recommend
-    /// it not be used in production environments. For local testing, supplying ["localhost:5000"]
-    /// as an argument for the local docker reigstry will allow for http connections to that registry.
+    /// Allows the host to pull actor and capability provider images from the supplied list
+    /// of valid OCI registries without using a secure (SSL/TLS) connection. This option is empty by default and we recommend
+    /// it not be used in production environments. For local testing, supplying `["localhost:5000"]`
+    /// as an argument for the local OCI registry will allow for http connections to that registry.
```

Comment:
The `localhost:5000` is how we have our `docker-compose` setup, but perhaps others won't know that context. What do you think about rephrasing the last sentence as something like:
```
For local testing, supplying `["localhost:5000"]` as an argument will allow for HTTP connections to a local OCI registry listening on port 5000.
```
Definitely will entertain some wording updates, but this may be a bit more explicit for the newbie that has their registry on port 420 without knowing

Reply:
going to remove this and just use the opportunity to give people a heads-up to make sure they supply the *correct* host name and port.